site/content/en/docs/Installation/Terraform/eks.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Installing Agones on AWS Elastic Kubernetes Service using Terraform"
+linkTitle: "AWS"
+weight: 20
+description: >
+  You can use Terraform to provision an EKS cluster and install Agones on it.
+---
+
+{{% feature expiryVersion="1.3.0" %}}
+This feature is under development and will be supported in the upcoming release.
+{{% /feature %}}
+
+{{% feature publishVersion="1.3.0" %}}
+## Installation
+
+### Creating Cluster
+
+You can use Terraform to provision your Amazon EKS (Elastic Kubernetes Service) cluster and install Agones on it using the Helm Terraform provider.
+
+The example of the EKS submodule config file can be found here:
+ {{< ghlink href="examples/terraform-submodules/eks/module.tf" >}}Terraform configuration with Agones submodule{{< /ghlink >}}
+
+Copy `module.tf` file into a separate folder.
+
+Configure your AWS CLI tool [CLI configure](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html):
+```
+aws configure
+```
+
+Configure your terraform:
+```
+terraform init
+```
+
+By editing `modules.tf` you can change the parameters that you need to, such as the EC2 instance type. Note that the maximum number of instances in the workers group is limited to 3.
+
+Now you can deploy Agones on EKS:
+```
+terraform apply
+```
+
+After deploying the cluster with Agones, you can get or update your kubeconfig by using:
+```
+aws eks --region us-west-2 update-kubeconfig --name agones-cluster
+```
+
+With the following output:
+```
+Added new context arn:aws:eks:us-west-2:601646756426:cluster/agones-cluster to /Users/user/.kube/config
+```
+
+Switch `kubectl` context to the recently created one:
+```
+kubectl config use-context arn:aws:eks:us-west-2:601646756426:cluster/agones-cluster
+```
+
+Check that you are authenticated against the recently created Kubernetes cluster:
+```
+kubectl get nodes
+```
+
+### Installing Agones
+
+You can use any option to [install Agones]({{< relref "../Install Agones/_index.md" >}}). For instance, Helm or YAML option.
+
+> Note: current EKS config does not contain Helm Terraform configuration as for other Cloud Providers. That's because of a known issue with AWS Terraform provider:
```

Comment:
Let's use the `alert` shortcode here to be consistent with the other alert notes in our documentation.

Reply:
Applied